components/Containers/ProgressionSidebar/ProgressionSidebarItem/index.module.css

```diff
@@ -0,0 +1,33 @@
+a.item {
```

Comment:
This should be just `.item`

Reply:
I tried but `a:link` precedences over `.classname` because of that I used `a.classname`.

Reply:
How does that even happen? Because, css modules should just be grabbing this as a classname, the `a.` should make no difference O.o

Reply:
I think there is something wrong with the CSS module. As you mentioned in the review, `ul` should not be styled at all under normal conditions, but when I checked, I noticed that it was styles written directly to global `ul` without using `:global` 🤔

Reply:
> I noticed that it was styles written directly to global ul without using :global 🤔

wuuuut... Do you mind giving a look into our CSS modules setup? @canerakdas?